### ISSUE TITLE
New modules can not be imported

### ISSUE THREAD

**Issue by saraedum** [nthiery/recursive-monkey-patch#5]
Currently, the following fails in the context of the documentation, i.e., when `submodule_new` is a module that has been introduced by the patch:
```
>>> import a_test_module.submodule_new
ModuleNotFoundError: No module named 'a_test_module.submodule_new'
```

**Comment by nthiery** [comment]
I guess this can be closed now?

**Comment by saraedum** [comment]
Yes, it closes automatically once you merge #6 :)
